src/resources/Pipelines/Pipelines.ts

```diff
@@ -9,17 +9,20 @@ import {
     PipelineModel,
     UpdatePipelineModel,
 } from './PipelinesInterfaces';
+import Statements from './Statements/Statements';

```

Comment:
```suggestion
import Statements from './Statements';
```

Reply:
`TS1192: Module '".../platform-client/src/resources/Pipelines/Statements/index"' has no default export.`

If I change to `import {Statements} from './Statements';` then I get `TS2305: Module '".../platform-client/src/resources/Pipelines/Statements/index"' has no exported member 'Statements'.`

I can make it work if I do 
```typescript
export class Statements extends Resource {
  ...
} 
export default Statements;
```
But I feel like the path in the import is cleaner

Reply:
If we look at all the other resources, we do
```ts
export default class Statements extends Resource {
    ...
}
```